src/NosCore.Packets/ServerPackets/CModePacket.cs

```diff
@@ -43,7 +43,10 @@ public class CModePacket : PacketDefinition
         [PacketIndex(5, IsOptional = true)]
         public byte MorphBonus { get; set; }
 
-        [PacketIndex(6, IsOptional = true)]
-        public byte? Size { get; set; }
+        [PacketIndex(6)]
+        public byte Size { get; set; }
+
+        [PacketIndex(7, IsOptional = true)]
+        public byte Unknown { get; set; }
```

Comment:
Size can't be null, Unknown will be optional because we don't know the use of it, normally its value is 0

Reply:
Size is not on cmode for monsters

Reply:
Also it’s optional mean can be null. So byte? Because byte will not be optional

Reply:
F**k, you are right, I didn't remember or know about it, srry.